tests/tests/test-class-admin.php

```diff
@@ -380,26 +377,18 @@ public function test_filter_role_caps() {
 	 * @requires PHPUnit 5.7
 	 */
 	public function test_ajax_filters() {
-		$user = new \WP_User( get_current_user_id() );
-
 		$this->_setRole( 'subscriber' );
 
+		$user = new \WP_User( get_current_user_id() );
+
 		$_POST['filter'] = 'user_id';
 		$_POST['q'] = $user->display_name;
 		$_POST['nonce'] = wp_create_nonce( 'stream_filters_user_search_nonce' );
 
 		$this->expectException( 'WPAjaxDieStopException' );
+		$this->expectExceptionMessage( '-1' );
```

Comment:
This is a simplification for the lines above to make it match the rest of the tests.